feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -184,9 +191,11 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
     this.afterCloseASMSaveCartDialog();
   }
 
-  clearText() {
-    this.cartId.setValue('');
-    this.resetDeeplinkCart();
+  clearText(): void {
```

Comment:
seems the only usage is in feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.html, and it already have feature flag in asm-bind-cart.component.html line22, so there is no need to add feature flag here.

Reply:
have been modified.